src/frontend/src/common/types/user.d.ts

```diff
@@ -18,4 +18,5 @@ interface User {
   name: string;
   systemAdmin: boolean;
   agreedToTos: boolean;
+  acknowledgedPolicyVersion: string; // Date column in DB, will be ISO 8601: YYYY-MM-DD
```

Comment:
This will just be serialized as '' if the version in the db is null?

Reply:
It comes through as JS `null` when DB NULL, so good point, I'll go back and add a union type